doc/user_manual/postprocessor.tex

```diff
@@ -780,7 +787,14 @@ \subsubsection{LimitSurfaceIntegral}
                optional field}, name of the distribution that is associated to this variable.
               Its name needs to be contained in the \xmlNode{Distributions} block explained
               in Section \ref{sec:distributions}. If this node is not present, the  \xmlNode{lowerBound}
-              and  \xmlNode{upperBound} XML nodes must be inputted.
+              and  \xmlNode{upperBound} XML nodes must be inputted. It requires the following two attributes:
+            \begin{itemize}
+              \item \xmlAttr{class}, \xmlDesc{required string attribute}, is the main
+              ``class'' the listed object is from, the only acceptable class for
+              this post-processor is \xmlString{Distributions};
```

Comment:
?

Reply:
are you sure about this?

Reply:
I have changed the source code to be consistent with other post-processors.

Reply:
okay